lua/ulx/modules/sh/cfc_refund.lua

```diff
@@ -0,0 +1,37 @@
+local CATEGORY_NAME = "Refund"
+
+local playerKills = {}
+
+hook.Add( "PlayerInitialSpawn", "OnPlayerJoin", function(ply)
+	playerKills[ply] = {}
+end)
+
+hook.Add( "PlayerDisconnected", "OnPlayerDisconnect", function(ply)
+	playerKills[ply] = nil
+
+	for k, killData in pairs( playerKills )
+		killData[ply] = nil
+	end
+end)
+
+hook.Add( "PlayerDeath", "OnPlayerKill", function( ply, inflictor, attacker )
```

Comment:
this hook names are not great
they are too generic and dont describe what the function does
e.g. `OnPlayerJoin` could  be changed to `CFC_ULXCommands_CreateDeathTable`